WordPress/gutenberg
# wp-env: Edit PHP configuration with htaccess from .wp-env.json.

## Description
Closes #28703 and #29430.  Related: #30377.

This PR makes it possible to edit PHP configuration via htaccess from .wp-env.json. 

wp-env.json
```json 
{
	"core": "WordPress/WordPress",
	"phpConfig": {
		"post_max_size": "4G",
		"upload_max_filesize": "4G"
	}
}
```

.httacess

```
# BEGIN wp-env
# The directives (lines) between "BEGIN wp-env" and "END wp-env" are
# dynamically generated, and should only be modified via WordPress filters.
# Any changes to the directives between these markers will be overwritten.
php_value post_max_size 4G
php_value upload_max_filesize 4G
# END wp-env
```


## How has this been tested?

1. Create .wp-env.override.json and start wp-env.

```
{
	"core": "WordPress/WordPress",
	"plugins": [
		"."
	],
	"phpConfig": {
		"post_max_size": "4G",
		"upload_max_filesize": "4G"
	},
	"env": {
		"tests": {
			"phpConfig": {
				"post_max_size": "6G",
				"upload_max_filesize": "6G"
			}
		}
	}
}
```

2. Check upload size and htaccess value.

## Types of changes
New feature 

## Checklist:
